samples/XCT.Sample/Resx/AppResources.Designer.cs

```diff
@@ -10,48 +10,35 @@
 
 namespace Xamarin.CommunityToolkit.Sample.Resx {
     using System;
+    using System.Reflection;
     
     
-    /// <summary>
-    ///   A strongly-typed resource class, for looking up localized strings, etc.
-    /// </summary>
-    // This class was auto-generated by the StronglyTypedResourceBuilder
```

Comment:
Please revert changer in this file. It's autogenerated, and those changes will be brought back automatically next time any resource is added, deleted or updated